### ISSUE TITLE
`blt local:setup` fails to add hostname entries in /etc/hosts

### ISSUE THREAD

**Issue by timwood** [acquia/blt#651]
Mac OS 10.11.6
BLT 8.5.2

When following the [Creating a new project](http://blt.readthedocs.io/en/8.x/readme/creating-new-project/) guide when you get to step 4 the `blt local:setup` command fails at setup:drupal:install due to hostname entries not being created in /etc/hosts. I have vagrant-hostsupdater (1.0.2) plugin.

Running `vagrant halt` `vagrant up` does not help, but running `vagrant provision` seems to add the proper entries to /etc/hosts.

```
blt > setup:drupal:install:

     [echo] Installing Drupal.
ssh: Could not resolve hostname local.authenticator.gov: nodename nor servname provided, or not known
[phingcall] /Users/USERNAME/Sites/d8/vendor/acquia/blt/phing/tasks/setup.xml:157:34: Drush exited with code 255

BUILD FAILED
/Users/USERNAME/Sites/d8/vendor/acquia/blt/phing/tasks/local-sync.xml:12:30: Execution of the target buildfile failed. Aborting.

Total time: 2.8696 seconds
```

**Comment by bobbygryzynger** [comment]
@timwood - I've actually run into this problem before, not using BLT, just Vagrant. 

It seems that the hostsupdater extension sometimes adds entries on the same line as existing entries. The best way I've found to guard against this is by adding a comment line between blocks of entries and by cleaning up entries in `/etc/hosts` that have been added to an existing line. 

It's a workaround but this issue is likely outside the scope of BLT.

**Comment by timwood** [comment]
@bobbygryzynger What does your `/etc/hosts` end up looking like when hostupdater adds entries on the same line as existing entries?

**Comment by bobbygryzynger** [comment]
@timwood - it ends up looking something like:
```
127.0.0.1       previous.local 192.168.88.88  vagrant.local  # VAGRANT: 2349c50d13fdba5e48c1018e52aae107 (vagrant) / 08ea9f46-9570-49b5-ae81-b835d63efb79
```

When it should look like:
```
127.0.0.1       previous.local 
192.168.88.88  vagrant.local  # VAGRANT: 2349c50d13fdba5e48c1018e52aae107 (vagrant) / 08ea9f46-9570-49b5-ae81-b835d63efb79
```

**Comment by timwood** [comment]
I don't see any evidence of this in my /etc/hosts.

**Comment by grasmash** [comment]
I've also encountered this issue. However, BLT is not responsible for modifications to `/etc/hosts`. The vagrant hostsupdater plugin should do this. BLT only verifies that it is installed. You can check this using `vagrant plugin list`. If it is not installed, please open an issue regarding that.

You may want to post an issue in the https://github.com/geerlingguy/drupal-vm/issues queue to further debug.

**Comment by dougvann** [comment]
I had the same error. I manually edited /etc/hosts using the vagrant_ip and the vagrant_hostname.
192.168.25.8    local.blt2.com
When I re-ran blt local:setup the process hung on installing Drupal then finally timed out.

blt > drupal:install:
     [echo] Installing Drupal...
ssh: connect to host local.blt2.com port 22: Operation timed out

I did check and vagrant hostupdater is installed.
Not sure what to do next.
